pkg/vm/vm_test.go

```diff
@@ -1321,6 +1321,16 @@ func TestTRY(t *testing.T) {
 			inner := getTRYProgram(throw, add5, []byte{byte(opcode.THROW)})
 			getTRYTestFunc(32, inner, add5, add9)(t)
 		})
+		t.Run("TryMaxDepth", func(t *testing.T) {
```

Comment:
I think this test actually fails because both `TRY` parameters are `0`.

Reply:
Well, turns out it isn't, but that's because we a checking `TRY` params incorrectly.
https://github.com/neo-project/neo-vm/blob/master/src/neo-vm/ExecutionEngine.cs#L1367
Maybe it can be fixed right here.

Reply:
Fixed that.